src/core/Microsoft.PowerToys.Settings.UI.Lib/ColorPickerSettings.cs

```diff
@@ -23,13 +23,7 @@ public ColorPickerSettings()
 
         public virtual void Save()
         {
-            // Save settings to file
-            var options = new JsonSerializerOptions
-            {
-                WriteIndented = true,
```

Comment:
Can you preserve the indentation?

Reply:
The JsonSerializerOptions are configured in SettingsUtils on line 50 :)

Reply:
I tested it and it doesn't indent the json in the file.

Reply:
I've tried debugging this issue and I started noticing something weird. The settings UI seems to use IPC with the runner which will reparse the json (over the IPC) and write the settings to disk (unformatted).

There are only a few specific situations where the PowerToys itself will write to the settings files (mainly when there is no file yet created).

Sending formatted json over the IPC does not result in a formatted save file.

The advantage of changing the write function is lower (as it only seems to be used when writing via the C# Plugins except the setting Ui)
Reading should benefit more as that is done via a file watcher and settings overload construction.

Still two small performance improvements!